spec/transifex_project_spec.rb

```diff
@@ -4,7 +4,14 @@
 include Txdb
 
 describe TransifexProject, test_db: true do
-  let(:database) { TestDb.database }
+  let(:database) do
+    TestDb.setup do
+      create_table(:foo) do
```

Comment:
It's a dummy table so `.setup` will create the database. Probably not necessary, I'll see about removing it since the table is never actually used.